portafly/src/react-app-env.d.ts

```diff
@@ -1,2 +1,69 @@
+/* eslint import/no-default-export: 0 */
+// / <reference types="node" />
```

Comment:
Watch out!, it's been formatted by eslint automatically and it will cause problems with TS. There should be no space: `/// <reference..`
I say we could esling-ignore the whole `/config` folder... if not, disable the rule in this file like it was before: https://github.com/3scale/porta/blob/4581223c9d11afeb04aace285ce8bcc8f7664612/portafly/src/react-app-env.d.ts#L1-L2